File: code-recipes/php/test/awd/InboundOrderCreationRecipeTest.php
<?php
namespace Test\awd;

use Src\awd\InboundOrderCreationRecipe;
use Test\util\RecipeTestCase;

class InboundOrderCreationRecipeTest extends RecipeTestCase
{

    protected function setUp(): void
    {
        $this->recipe = new InboundOrderCreationRecipe();
        $this->responses = [
            "awd-checkInboundEligibility",
            "awd-createInbound",
            "awd-getInbound",
            ""
        ];
    }
}

File: code-recipes/java/src/test/java/awd/InboundOrderCreationRecipeTest.java
package awd;

import util.RecipeTest;

import java.util.List;

public class InboundOrderCreationRecipeTest extends RecipeTest {

    protected InboundOrderCreationRecipeTest() {
        super(
                new InboundOrderCreationRecipe(),
                List.of(
                        "awd-checkInboundEligibility",
                        "awd-createInbound",
                        "awd-getInbound",
                        ""
                )
        );
    }
}
